library_generation/gapic-generator-java-wrapper

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
```

Comment:
Can you add a short comment explaining the purpose of this wrapper?

Reply:
done